src/TypeFormatter/UnionTypeFormatter.ts

```diff
@@ -15,11 +17,51 @@ export class UnionTypeFormatter implements SubTypeFormatter {
         return type instanceof UnionType;
     }
     public getDefinition(type: UnionType): Definition {
+        // FIXME: Filtering never types from union types is wrong. However,
```

Comment:
Can you file an issue for this? A fixme in the code here is too easy to overlook.

Reply:
I was incorrect, filtering never types from unions is valid. However, I thought this statement would be superfluous as never types are filtered upon construction of the union. Apparently, this is not always the case, as the `hidden` tests point out.